scwx-qt/source/scwx/qt/ui/setup/audio_codec_page.cpp

```diff
@@ -167,7 +167,7 @@ bool AudioCodecPage::IsRequired()
    // Setup is required if codec errors are not ignored, and the default codecs
    // are not supported
    return (!ignoreCodecErrors &&
-           oggCodecs.contains(QMediaFormat::AudioCodec::Vorbis));
+           !oggCodecs.contains(QMediaFormat::AudioCodec::Vorbis));
```

Comment:
While this is the proper logic, I'm not sure checking for Vorbis was correct to begin with.

Running with this change on my system causes the setup dialog to display.  The purpose of this was to ensure that the default audio alert is playable (Ogg Vorbis).  Even though oggCodecs doesn't contain Vorbis on my Windows system (only FLAC and WMA), the Ogg Vorbis file still plays (scwx-qt/res/audio/wikimedia/Emergency_Alert_System_Attention_Signal_20s.ogg).

I think it might be better here to just check that oggCodecs isn't empty.

Reply:
Makes sense - it implies that some Ogg support exists.